spec/unit/peep_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'peep'
+
+describe Peep do
+
```

Comment:
I like the use of `change`, `include` and `start_with` matchers in this set of tests, which provide good coverage of what a peep is supposed to contain.